_posts/en/newsletters/2021-04-07-newsletter.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Bitcoin Optech Newsletter #143'
+permalink: /en/newsletters/2021/04/07/
+name: 2021-04-07-newsletter
+slug: 2021-04-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains our regular sections with announcements
+of new releases and release candidates, plus notable changes to popular
+Bitcoin infrastructure projects.
+
+## News
+
+*No noteworthy news to report this week.*
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+
+- [C-Lightning 0.10.0][C-Lightning 0.10.0] is the newest major release
+  of this LN node software. It contains a number of enhancements to its
+  API and includes experimental support for [dual funding][topic dual
+  funding].
+
+- [BTCPay 1.0.7.2][] fixes minor issues discovered after last
+  week's security release.
+
+<!-- FIXME:harding to update tuesday -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20286][] removes the fields `addresses` and `reqSigs` from the
+  responses of the RPCs `gettxout`, `getrawtransaction`,
+  `decoderawtransaction`, `decodescript`, `gettransaction`, and the REST
+  endpoints `/rest/tx`, `/rest/getutxos`, `/rest/block`. When a well-defined
+  address exists, the responses now include the optional field `address`
+  instead. The deprecated fields had been used in the context of bare multisig
+  which has no substantial use on the network today. The deprecated fields can
+  be output via the configuration option `-deprecatedrpc=addresses` until the
+  option will get removed in Bitcoin Core 23.0.
```

Comment:
s/until the option will get removed/until the option is removed/

Reply:
sgtm, changed